Directory.Build.props

```diff
@@ -12,7 +12,7 @@
 
     <!-- Set this property to false if you don't want to use the runtime
           framework version defined in Versions.props -->
-    <UseCustomRuntimeVersion>true</UseCustomRuntimeVersion>
+    <UseCustomRuntimeVersion>false</UseCustomRuntimeVersion>
```

Comment:
I think either `true` or `false` could be a good default here. `true` is good if you're starting an experiment during a release or are upstreaming features during the experiment (instead of only at the end). `false` is good if you're only going to build on released versions of .NET. I don't care what the default is.